atk/Object.custom

```diff
@@ -61,16 +61,11 @@
 
 		static RefChildDelegate RefChildCallback;
 
-		[DllImport("libgobject-2.0-0.dll", CallingConvention=CallingConvention.Cdecl)]
-		static extern IntPtr g_object_ref (IntPtr handle);
-
 		static IntPtr RefChild_cb (IntPtr raw, int i)
 		{
 			try {
 				Atk.Object obj = GLib.Object.GetObject (raw, false) as Atk.Object;
 				Atk.Object child = obj.OnRefChild (i);
```

Comment:
@alanmcgovern you are talking specifically about this.

The thing is, we shouldn't be ref-ing unconditionally. If the managed implementation wants to ref something, it should do it itself, or let the base method handle it. This probably isn't bound with a base call (in OnRefChild) because the native implementation is `null`.

https://github.com/GNOME/atk/blob/08b16a21c004e5663b3e7362fcf6c3ff981d9f43/atk/atkobject.c#L344